code/modules/mob/living/carbon/xenomorph/Powers.dm

```diff
@@ -1432,6 +1450,30 @@
 	playsound(loc, "sound/bullets/acid_impact1.ogg", 25)
 
 
+/mob/living/carbon/Xenomorph/proc/acid_check(obj/effect/xenomorph/acid/new_acid, obj/effect/xenomorph/acid/current_acid)
+	if(new_acid.acid_strength < current_acid.acid_strength)
```

Comment:
Could also sanity check to see if both parameters exist otherwise it may runtime.